crates/wg_tests/src/test_fragments.rs

```diff
@@ -86,25 +92,29 @@ pub fn generic_fragment_drop<T: Drone + Send + 'static>() {
     // "Client" sends packet to the drone
     d_send.send(msg.clone()).unwrap();
 
-    let dropped = Nack {
-        fragment_index: 1,
-        nack_type: NackType::Dropped,
-    };
-    let srh = SourceRoutingHeader {
-        hop_index: 1,
-        hops: vec![11, 1],
-    };
-    let nack_packet = Packet {
-        pack_type: PacketType::Nack(dropped),
-        routing_header: srh,
-        session_id: 1,
-    };
+    let nack_packet = Packet::new_nack(
+        SourceRoutingHeader {
+            hop_index: 1,
+            hops: vec![11, 1],
```

Comment:
Should be 12, 11 1

Reply:
Are you sure? In this case the packet is being dropped by 11, it did not reach 12